Classes/Utility/FLEXRuntimeUtility.m

```diff
@@ -277,14 +277,20 @@ + (NSArray *)prettyArgumentComponentsForMethod:(Method)method
     NSMutableArray *components = [NSMutableArray array];
     
     NSString *selectorName = NSStringFromSelector(method_getName(method));
-    NSArray *selectorComponents = [selectorName componentsSeparatedByString:@":"];
-    unsigned int numberOfArguments = method_getNumberOfArguments(method);
+    NSMutableArray *selectorComponents = [[selectorName componentsSeparatedByString:@":"] mutableCopy];
+    if ([selectorComponents.lastObject isEqualToString:@""]) {
+        [selectorComponents removeLastObject];
+    }
+    
+    if (selectorComponents.count == 1) {
```

Comment:
Will this change the return value for single argument methods?

Reply:
Yes, it does. His test did not accurately reflect this, as he adjusted the test to account for his own change.